sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/configurationproperties/models/MessageProperties.java

```diff
@@ -26,4 +32,15 @@ public static MessageProperties mapToMessageProperties(Exchange exchange) {
     messageProperties.setHeaders(filterNonSerializableHeaders(exchange));
     return messageProperties;
   }
+
+  @SneakyThrows
+  public String getBody() {
+    if (isNotBlank(body) && body.startsWith(FILE_PREXIX)) {
+      String bodyLocation = body.substring(FILE_PREXIX.length());
+      body =
+          FileUtils.readFileToString(
+              new ClassPathResource(bodyLocation).getFile(), StandardCharsets.UTF_8);
+    }
+    return body;
+  }
```

Comment:
FILE_PREFIX is definitely unnecessary and will be replaced.

My idea was to make the expression as simple as possible and meaningful for non-tech users. That's why I wanted to avoid classpath: and I was encouraged by the idea that tests make most sense if delivered as part of the source code.

Generally, for any more precisely and technical features I would use headers because tech roles wouldn't mind it and basic "language" remains simple and clear.